docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  mockify:
+    build: .
+    volumes:
+      - ./app:/app
+    ports:
+      - "8001:8001"
```

Comment:
The only thing I would suggest here is to see if it's possible to use an environment variable for the port. In case you needed to spin up several different containers on different ports. It may not be possible though.